### DISCUSSION TITLE
Show new notifications icon on watch face

### DISCUSSION THREAD

**Discussion by 31337-4554551n** [InfiniTimeOrg/InfiniTime#1615] [Ideas]
### 

- [X] I searched for similar feature request and found none was relevant.

### Pitch us your idea!

Show new notifications icon on watch face

### Description

The idea would be to have a counter on the watch face for missed notifications. It could reset to 0 when the notifications is swiped up to, or ideally when they are dismissed off the phone.

Even better would be 4 notification icons:
email
missed calls
sms
other
That each increment as notifications come through, and are reset as they are cleared off the phone.

**Comment by geekbozu** [comment]
So there is a "notification" icon when the watch is in silent mode, 
I would like to see a revamp to the notification system as well so we can get more real time information about the notifications currently on the watch and such as well. 
#781 is something else that needs doing in this same vain.
